### PR TITLE
[CNVS Upgrade] Enable media-object styles

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

File: src/styles/index.less [hunk 1]
```diff
@@ -48,10 +48,14 @@
...
+// Iframes
 
-@import 'layout/sidebar/variables.less';
-@import 'layout/sidebar/styles.less';
+@import 'layout/iframes/styles.less';
+
+// Media Object
+
+@import 'layout/media-object/variables.less';
+@import 'layout/media-object/styles.less';
```
Comment (mlunoe) [inline src/styles/index.less]: This file is still under `layout/media-object.less`

File: src/styles/index.less [hunk 2]
```diff
@@ -61,9 +65,15 @@
 
 @import 'layout/page-body/styles.less';
 
-// @import 'layout/iframes.less';
-// @import 'layout/media-object.less';
-// @import 'layout/modal.less';
+// Responsive Utilities
+
+@import 'layout/responsive-utilities/styles.less';
```
Comment (mlunoe) [inline src/styles/index.less]: This file needs to be moved from `utilities/responsive-utilities.less`

File: src/styles/index.less [hunk 3]
```diff
@@ -48,10 +48,14 @@
 
 @import 'layout/header/styles.less';
 
-// Sidebar
+// Iframes
 
-@import 'layout/sidebar/variables.less';
-@import 'layout/sidebar/styles.less';
+@import 'layout/iframes/styles.less';
```
Comment (mlunoe) [inline src/styles/index.less]: This file needs to be moved from `layout/iframes.less`

Comment (jfurrow) [inline src/styles/index.less]: I lost a commit somewhere... this is fixed!

File: src/styles/index.less (hunk 2 above)
Comment (jfurrow) [inline src/styles/index.less]: I lost a commit somewhere... but this actually should have been removed. The responsive classes that we use (`.hidden-*`, `.visible-*` etc) are coming directly from CNVS right now, though they are mis-named and need to be fixed there. I spoke with John A about this already, it just needs to be done.

**Pull request merged**